modules/ROOT/pages/troubleshooting-visualization.adoc

```diff
@@ -0,0 +1,87 @@
+= Troubleshooting Visualization
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+3
+
+== About the Troubleshooting Visualization
+
+The Troubleshooting Visualization shows you the current and past health of your Mule apps. It allows you to quickly visualize where your network is slow, receiving heavy traffic or failing. From this visualization, you can drill down directly into Anypoint Monitoring dashboards or logs to continue to troubleshoot the performance of your systems.
+
+== Choosing your Metric
+
+Use the options in the *Metric* card to select the metric that you want to show for all the nodes in your application network. For applicable metrics, the metric value will also appear on the connection, or _edge_.
+
+In the *Metrics* section, you can select one of the following metrics:
+
+* *Avg response time*
+* *Avg throughput*
+* *Failure rate*
+* *Avg CPU utilization*
+* *Avg memory utilization*
+
+For the selected time range, _failure rate_ is equal to the total number of failures divided by the total number of requests.
+
+For average response time, average throughput, and failure rate, you can select *Inbound* or *Outbound* metrics.
+If you select inbound, the value that appears on the application is the weighted average of that particular metric for all inbound connections to that node.
+If you select outbound, the value is the weighted average of that metric for all outbound connections.
+
+In the *Time Range* section, you can select the time window used to calculate the average values of the selected metric.
+
+=== Metrics Notation
+
+To avoid long numbers, Anypoint Visualizer uses unit prefixes to simplify your view. For example:
```

Comment:
```suggestion
To avoid long numbers, Anypoint Visualizer uses unit prefixes to simplify your view:
```
However, I don't understand what is shown. Is this three abbreviations for the same value?
Also...do we really have to tell users about this in the context of choosing metrics?